### PR TITLE
RavenDB-17980 Cleanup topology files for TestDriver

### PR DESCRIPTION
**Pull request by YonatanResh**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-17980/Cleanup-topology-files-for-TestDriver

### Testing 

- Tests have been added that prove the fix is effective or that the feature works
- It has been verified by manual testing

### CODE REVIEW

File: src/Raven.TestDriver/RavenTestDriver.cs [hunk 1]
```diff
@@ -88,6 +88,8 @@ protected IDocumentStore GetDocumentStore(GetDocumentStoreOptions options = null
 
             PreInitialize(store);
 
+            store.Conventions.DisableTopologyCache = true;
```
Comment (ppekrol) [inline src/Raven.TestDriver/RavenTestDriver.cs]: let's set it up few lines up in place where are we creating a document store. This way you are allowing user to turn it on.

File: src/Raven.TestDriver/RavenTestDriver.cs [hunk 2]
```diff
@@ -278,6 +280,8 @@ private static IDocumentStore RunServer()
                 Urls = new[] { url.AbsoluteUri }
             };
 
+            store.Conventions.DisableTopologyCache = true;
```
Comment (ppekrol) [inline src/Raven.TestDriver/RavenTestDriver.cs]: inline it?

            var store = new DocumentStore
            {
                Urls = new[] { url.AbsoluteUri },
                Conventions = {
                    DisableTopologyCache  = true
                }
            };

File: src/Raven.TestDriver/RavenTestDriver.cs (hunk 1 above)
Comment (YonatanResh) [reply src/Raven.TestDriver/RavenTestDriver.cs]: done.

File: src/Raven.TestDriver/RavenTestDriver.cs (hunk 2 above)
Comment (YonatanResh) [reply src/Raven.TestDriver/RavenTestDriver.cs]: done

**Pull request merged**